tools/internal_ci/macos/grpc_basictests_dbg.cfg

```diff
@@ -27,5 +27,5 @@ action {
 
 env_vars {
   key: "RUN_TESTS_FLAGS"
-  value: "-f basictests macos dbg --internal_ci -j 1 --inner_jobs 4 --bq_result_table aggregate_results"
+  value: "-f basictests macos dbg --exclude objc --internal_ci -j 1 --inner_jobs 4 --bq_result_table aggregate_results"
```

Comment:
addition of this `--exclude objc` is unrelated to the timing instrumentation done elsewhere, right?

Reply:
You're right. I'll remove from this PR.

Reply:
Done. PTAL.